Python/compile.c

```diff
@@ -72,18 +72,20 @@
 
 /* Pseudo-instructions used in the compiler,
  * but turned into NOPs by the assembler. */
-#define SETUP_FINALLY 255
-#define SETUP_CLEANUP 254
-#define SETUP_WITH 253
-#define POP_BLOCK 252
-#define JUMP 251
+#define SETUP_FINALLY 256
```

Comment:
Just an idea, but would it make sense for the virtual opcodes to be negative?
>0: Real
==0: Cache
<0: Virtual 
It seems like a simpler scheme.